google/cloud/logging_v2/_instrumentation.py

```diff
@@ -67,8 +67,12 @@ def _create_diagnostic_entry(name=_PYTHON_LIBRARY_NAME, version=_LIBRARY_VERSION
             _INSTRUMENTATION_SOURCE_KEY: [_get_instrumentation_source(name, version)]
         }
     }
-    kw["severity"] = "INFO"
-    entry = StructEntry(payload=payload, **kw)
+    active_kws = {"severity": "INFO"}
+    # only keep the log_name and resource from the parent log
+    for key in ["log_name", "resource"]:
```

Comment:
@losalex Are there any other fields that should be included? Do the other languages keep the resource field?

Reply:
Those are mandatory ones and those are the only ones which should be included.